lib/host/events/submit_all_orders.js

```diff
@@ -3,6 +3,8 @@
 const _isObject = require('lodash/isObject')
 const _omit = require('lodash/omit')
 const _isFunction = require('lodash/isFunction')
+const _isString = require('lodash/isString')
+const isObject = require('lodash/isObject')
```

Comment:
for consistency maybe rename `isObject` to `_isObject`

Reply:
fixed